invenio_vocabularies/contrib/awards/awards.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Vocabulary awards."""
+
+from invenio_db import db
+from invenio_pidstore.models import PIDStatus
+from invenio_records.dumpers import ElasticsearchDumper
+from invenio_records.systemfields import ModelField, RelationsField
+from invenio_records_resources.factories.factory import RecordTypeFactory
+from invenio_records_resources.records.systemfields import ModelPIDField, \
+    PIDListRelation
+from ...services.permissions import PermissionPolicy
+from sqlalchemy_utils.types import ChoiceType
+from ..funders.api import Funder
+from .config import AwardsSearchOptions, service_components
+from .schema import AwardSchema
+
+award_relations = RelationsField(
+    funders=PIDListRelation(
```

Comment:
comment: Im not a 100% sure this is gonna work with the new `ModelPIDField`, since the `pid_field` is now a `PersistentIdentifierWrapper` and it might be expected to have some funcs that are not available. TBD.

Reply:
Updated the relation config, and tests for it, but it is working 🚀